docs/en/interfaces/cli.md

```diff
@@ -158,6 +158,116 @@ $ clickhouse-client --param_tuple_in_tuple="(10, ('dt', 10))" -q "SELECT * FROM
 $ clickhouse-client --param_tbl="numbers" --param_db="system" --param_col="number" --query "SELECT {col:Identifier} FROM {db:Identifier}.{tbl:Identifier} LIMIT 10"
 ```
 
+## Connection string {#connection_string}
+
+The connection string for clickhouse-client is presented in URI format:
+
+```text
+clickhouse://[user_info@][hosts_and_ports][/dbname][?query_parameters]
```

Comment:
Here is URI RFC:
https://datatracker.ietf.org/doc/html/rfc3986

The document uses `userinfo`. Also, `userspec` and `hostspec` make spellcheck raise errors in documentation. I can add exceptions for these cases and use the words you suggested.

Reply:
`*spec` is the term used by Postgres (which I trust most in terms of "doing-the-right-think" as a DB). But I don't mind too much.

Another option is to simply omit the term `user_info` and write out the actual syntax elements: `user[:password]@`. Maybe that creates the least confusion.

Reply:
Ok, lets use: `user[:password]@` instead of `user_info@` in the documentation.

Keep `hosts_and_ports` and `query_parameters`.  
Rename `dbname` --> `database`

Reply:
Done